examples/basic_ops.rs

```diff
@@ -1,12 +1,12 @@
 #![allow(non_snake_case)]
 extern crate rand;
 /// The purpose of this implementation is to provide support for one of the most
-/// commonly used operations over EC which is Random Scalar Mul.
+/// commonly used operations in Elliptic Curve Cryptography (ECC), which is Random Scalar Mul.
```

Comment:
Changing this is correct but we should refactor this example since the API now provides an improved methodology to get random points (From Elligator-ristretto-flavour).

Reply:
Okay, I can do the API change